pages/dkp/kommander/2.3/clusters/custom-domain-certificate/configuration/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: layout.pug
+navigationTitle: Configuration 
+title: Configure a custom domain and certificate for your cluster
+menuWeight: 20
+excerpt: Configure a custom domain and certificate in the Management and any Managed or Attached Clusters
+beta: false
+enterprise: false
+---
+
+This section describes how to enable custom domains and certificates in your **Management and any Managed or Attached** clusters after the installation of DKP. If you want to set up a custom domain and certificate for your Management Cluster **during the installation**, refer to the [Customize a domain and certificate during installation][management] documentation.
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that will allow Kommander to implement the established adjustments on top of the default or any previous configuration you have made.
+
+## Configure custom domains or custom certificates
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster of the environment where you want to customize the domain or certificate by setting the <code>KUBECONFIG=<path></code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both. For this, refer to the following examples:
+
+1.  You have two options to create or update and apply the `KommanderCluster` object with the wanted ingress. Remember to specify the cluster in the `kubeconfigRef` name field.
+
+    One option is to use a certificate that is managed automatically by cert-manager with the ACME protocol like Let's Encrypt. For this, reference the `Issuer` or `ClusterIssuer` **to be used by cert-manager** in the `issuerRef` field, and enter the custom domain in the `hostname` field of the target cluster:
```

Comment:
> with the ACME protocol like Let's Encrypt
Any issuer supported by cert-manager is supported. ACME is the most common case, but not the only possibility.